desktop/core/src/desktop/management/commands/rungunicornserver.py

```diff
@@ -169,17 +171,89 @@ def load(self):
     return self.load_wsgiapp()
 
 
+def fetch_ipv6_bind_address(http_host, http_port):
```

Comment:
Is it possible to move this function to some utils file? Basically to keep rungunicornserver.py clean and short. I am thinking of how @ranade1 created a tls_utils.py in https://github.com/cloudera/hue/pull/4237.